<?xml version='1.1' encoding='UTF-8'?>
<Tag plugin="workflow-support@3.3">
  <node class="cps.n.StepAtomNode" plugin="workflow-cps@2.67">
    <parentIds>
      <string>34</string>
    </parentIds>
    <id>35</id>
    <descriptorId>org.jenkinsci.plugins.workflow.steps.durable_task.BatchScriptStep</descriptorId>
  </node>
  <actions>
    <cps.a.ArgumentsActionImpl plugin="workflow-cps@2.67">
      <arguments>
        <entry>
          <string>script</string>
          <string>@&quot;C:\Program Files (x86)\Jenkins\..\Jenkins_Backup\20190503_154555\git.bat&quot;</string>
        </entry>
        <entry>
          <string>label</string>
          <string></string>
        </entry>
      </arguments>
      <isUnmodifiedBySanitization>true</isUnmodifiedBySanitization>
    </cps.a.ArgumentsActionImpl>
    <wf.a.TimingAction plugin="workflow-api@2.33">
      <startTime>1556898388599</startTime>
    </wf.a.TimingAction>
    <s.a.LogStorageAction/>
    <wf.a.ErrorAction plugin="workflow-api@2.33">
      <error class="org.jenkinsci.plugins.workflow.steps.FlowInterruptedException" plugin="workflow-step-api@2.19">
        <stackTrace>
          <trace>org.jenkinsci.plugins.workflow.cps.CpsFlowExecution.interrupt(CpsFlowExecution.java:1137)</trace>
          <trace>org.jenkinsci.plugins.workflow.job.WorkflowRun$2$1.run(WorkflowRun.java:361)</trace>
          <trace>jenkins.security.ImpersonatingScheduledExecutorService$1.run(ImpersonatingScheduledExecutorService.java:58)</trace>
          <trace>java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)</trace>
          <trace>java.util.concurrent.FutureTask.run(Unknown Source)</trace>
          <trace>java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(Unknown Source)</trace>
          <trace>java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)</trace>
          <trace>java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)</trace>
          <trace>java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)</trace>
          <trace>java.lang.Thread.run(Unknown Source)</trace>
        </stackTrace>
        <suppressedExceptions class="java.util.Collections$UnmodifiableRandomAccessList" resolves-to="java.util.Collections$UnmodifiableList">
          <c class="list"/>
          <list reference="../c"/>
        </suppressedExceptions>
        <result>
          <name>ABORTED</name>
          <ordinal>4</ordinal>
          <color>ABORTED</color>
          <completeBuild>false</completeBuild>
        </result>
        <causes class="java.util.Arrays$ArrayList">
          <a class="jenkins.model.CauseOfInterruption-array">
            <jenkins.model.CauseOfInterruption_-UserInterruption>
              <user>admin</user>
            </jenkins.model.CauseOfInterruption_-UserInterruption>
          </a>
        </causes>
      </error>
    </wf.a.ErrorAction>
  </actions>
</Tag>